Testing System/src/main/java/com/epam/practice/testingsystem/data/dao/DataParse.java

```diff
@@ -0,0 +1,158 @@
+package com.epam.practice.testingsystem.data.dao;
+
+import com.epam.practice.testingsystem.data.dto.*;
```

Comment:
Please do not use wildcards in the import section.

The automatic folding the imports could be disabled in the IDE. Alternatively, the threshold could be set (fold after 20+ imports).